src/library/rekordbox/rekordboxfeature.cpp

```diff
@@ -0,0 +1,946 @@
+// rekordboxfeature.cpp
+// Created 05/24/2019 by Evan Dekker
+
+#include <QMap>
+#include <QMessageBox>
+#include <QSettings>
+#include <QStandardPaths>
+#include <QtDebug>
+
+#include "library/rekordbox/rekordbox_pdb.h"
+#include "library/rekordbox/rekordboxfeature.h"
+
+#include "library/library.h"
+#include "library/librarytablemodel.h"
+#include "library/missingtablemodel.h"
+#include "library/queryutil.h"
+#include "library/trackcollection.h"
+#include "library/treeitem.h"
+#include "track/keyfactory.h"
+#include "util/db/dbconnectionpooled.h"
+#include "util/db/dbconnectionpooler.h"
+#include "util/file.h"
+#include "util/sandbox.h"
+#include "waveform/waveform.h"
+
+#include "widget/wlibrary.h"
+#include "widget/wlibrarytextbrowser.h"
+
+#define IS_RECORDBOX_DEVICE "::isRecordboxDevice::"
+#define IS_NOT_RECORDBOX_DEVICE "::isNotRecordboxDevice::"
+
+namespace {
+const QString kPDBPath = "PIONEER/rekordbox/export.pdb";
+const QString kPLaylistPathDelimiter = "-->";
+
+void clearTable(QSqlDatabase& database, QString tableName) {
```

Comment:
These were initially private members, however changed to address review https://github.com/mixxxdj/mixxx/pull/2119#discussion_r287557531

Reply:
I think they should either be wrapped in an anonymous namespace or made private members of RekordboxFeature. If they don't need state from the RekordboxFeature class, they don't need to be part of it, but they are logically related so it would make sense to make them members. I don't care much either way. :man_shrugging: